app/controllers/recipes_controller.rb

```diff
@@ -10,6 +10,7 @@ def index
 
   def show
     @recipe = Recipe.find(params[:id])
+    @ingredients = @recipe.recipe_ingredients
```

Comment:
I think we should rename this `recipe_ingredients` to just `ingredients`. It looks weird now - your choice :)